exporters/otlp/otlp.go

```diff
@@ -210,7 +211,7 @@ func (e *Exporter) Stop() error {
 
 // Export implements the "go.opentelemetry.io/otel/sdk/export/metric".Exporter
 // interface. It transforms metric Records into OTLP Metrics and transmits them.
-func (e *Exporter) Export(ctx context.Context, cps metricsdk.CheckpointSet) error {
+func (e *Exporter) Export(ctx context.Context, _ *resource.Resource, cps metricsdk.CheckpointSet) error {
```

Comment:
This is going to conflict with https://github.com/open-telemetry/opentelemetry-go/pull/626